docs/manual/map-key-checker.tex

```diff
@@ -12,13 +12,27 @@
 \sunjavadoc{java.base/java/util/Map.html\#get(java.lang.Object)}{\code{Map.get}}
 more precisely by refining its result to \<@NonNull> in some cases.
 
+
+\sectionAndLabel{Invoking the Map Key Checker}{map-key-invoking}
+
 You will not typically run the Map Key Checker.  It is automatically run by
 other checkers, in particular the Nullness Checker.
 
 You can unsoundly suppress warnings related to map keys with
 \<@SuppressWarnings("keyfor")>, or everywhere by using command-line option
 \<-AsuppressWarnings=keyfor>; see \chapterpageref{suppressing-warnings}.
 
+The command-line argument \<-AassumeKeyFor> makes the Map Key Checker
```

Comment:
Other than checking explicit KeyFor annotations, there isn't a need to run KeyFor Checker if this option is used.  What do you think about not running the KeyFor Checker if `-AassumeKeyFor` is passed?  (You can just set the KeyForAnnotatedTypeFactory to null in the Nullness Checker.)